torchao/quantization/quant_primitives.py

```diff
@@ -15,7 +15,7 @@
 
 
 _AFTER_TORCH_2_4_ONLY = [
-    "per_token_dynamic_quant",
+    "per_token_fake_dynamic_quant",
```

Comment:
this is used by executorch right now: https://github.com/pytorch/executorch/blob/main/examples/models/llama2/quantize.py#L16-L21, so please revert this change for now so we don't need to change executorch